hashicorp/dev-portal
# Remove Waypoint CLI docs

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-heat-chorewaypoint-ia-remove-cli-docs-hashicorp.vercel.app/waypoint) 🔎
- [Asana task](https://app.asana.com/0/1206225787851796/1206707955223701/f) 🎟️

## 🗒️ What

- Removed CLI from:
  - top level nav
  - product landing page sidebar nav
  - CLI card removed from the [Docs landing page](https://developer.hashicorp.com/waypoint/docs)
- Removed `pages/waypoint/commands`
- Add redirect to the [commands directory](https://github.com/hashicorp/waypoint/tree/main/website/content/commands) in archived repo ([PB](https://docs.google.com/document/d/1B1hIZYYaTtFAcudPqUY3tp8RAuJmtMm82rZ4izvQLiY/edit) redirect)
- Remove `"commands"` from Waypoint rootDocsPath

## 🤷 Why

- Part of Waypoint IA restructuring

## 🧪 Testing

- Visit [Waypoint landing page](https://dev-portal-git-heat-chorewaypoint-ia-remove-cli-docs-hashicorp.vercel.app/waypoint)
- Should match screenshot
<img width="1096" alt="Screenshot 2024-02-27 at 17 47 30" src="https://github.com/hashicorp/dev-portal/assets/55773810/52ad0c61-c4c4-4bcc-8853-83d771404eab">


- Visit [Waypoint docs page](https://dev-portal-git-heat-chorewaypoint-ia-remove-cli-docs-hashicorp.vercel.app/waypoint/docs)
- should not have CLI card per screenshot below
<img width="1098" alt="Screenshot 2024-02-27 at 17 48 26" src="https://github.com/hashicorp/dev-portal/assets/55773810/f3dd7d46-2543-4707-98f9-7f56914acc5f">


- Visiting [Waypoint CLI URL](https://dev-portal-git-heat-chorewaypoint-ia-remove-cli-docs-hashicorp.vercel.app/waypoint/commands) should redirect to [commands directory in GitHub](https://github.com/hashicorp/waypoint/tree/main/website/content/commands)
## 💭 Anything else?
nay
